### PR TITLE
UX Blitz

### PR DESCRIPTION
**Pull request by JakeHartnell**
Working on some quick and easy things to improve UX until the designs are ready.

### CODE REVIEW

**Review by ebaker — APPROVED** [review]
lgtm 👍 

thanks for the updates 🙂

File: .env.example [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 NEXT_PUBLIC_CHAIN_NAME=Wasmd Localhost
 NEXT_PUBLIC_CHAIN_BECH32_PREFIX=wasm
-NEXT_PUBLIC_CHAIN_RPC_ENDPOINT=http://localhost:3000/api
+NEXT_PUBLIC_CHAIN_RPC_ENDPOINT=http://localhost:3001/api
 NEXT_PUBLIC_CHAIN_REST_ENDPOINT=http://localhost:1317
+NEXT_PUBLIC_FEE_DENOM="ucosm"
 NEXT_PUBLIC_STAKING_DENOM="ustake"
-NEXT_PUBLIC_SITE_TITLE=CW-DAO
-NEXT_PUBLIC_SITE_ICON_URL="/juno.svg"
+NEXT_PUBLIC_SITE_TITLE=DAO DAO
```
Comment (gavindoughtie) [inline .env.example:8]: Need to quote these strings?

File: components/Layout.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ export default function Layout({ children }: { children: ReactNode }) {
     <div className="flex flex-col items-center justify-center min-h-screen bg-base-100 text-base-content">
       <Head>
         <title>{PUBLIC_SITE_TITLE}</title>
-        <link rel="icon" href="/juno.svg" />
+        <link rel="icon" href="/yin_yang.png" />
```
Comment (gavindoughtie) [inline components/Layout.tsx:13]: Hope we can go back to svg for release

**Review by gavindoughtie — APPROVED** [review]

File: components/Layout.tsx (hunk 2 above)
Comment (JakeHartnell) [reply components/Layout.tsx:13]: https://github.com/DA0-DA0/cw-dao-dapp/issues/34

File: .env.example (hunk 1 above)
Comment (JakeHartnell) [reply .env.example:8]: Don't think so, seems to work fine. :shrug:

**Pull request merged**